starter-core/src/test/groovy/io/micronaut/starter/feature/dependencies/MultipleSourcesDependencyResolver.groovy

```diff
@@ -4,16 +4,20 @@ import io.micronaut.context.annotation.Replaces
 import io.micronaut.core.io.ResourceResolver
 import io.micronaut.starter.build.dependencies.Coordinate
 import io.micronaut.starter.build.dependencies.CoordinatesUtils
+import io.micronaut.starter.build.dependencies.PomCoordinatesUtils
 import io.micronaut.starter.build.dependencies.PomDependencyVersionResolver
 import jakarta.inject.Singleton
 
 @Replaces(PomDependencyVersionResolver.class)
 @Singleton
-class MultiplePomDependencyResolver implements PomDependencyVersionResolver {
+class MultipleSourcesDependencyResolver implements PomDependencyVersionResolver {
```

Comment:
please, keep the old name to avoid an unnecessary breaking change.

Reply:
I don't understand what breaking change you're talking about : this is a test.

Reply:
I did not realise was a test. Sorry for the noise.